apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/MapEntryExpectationSamples.kt

```diff
@@ -0,0 +1,76 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples
+
+import ch.tutteli.atrium.api.infix.en_GB.key
+import ch.tutteli.atrium.api.infix.en_GB.toEqual
+import ch.tutteli.atrium.api.infix.en_GB.toEqualKeyValue
+import ch.tutteli.atrium.api.infix.en_GB.value
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class MapEntryExpectationSamples {
+
+    @Test
+    fun toEqualKeyValue() {
+        expect(mapOf(1 to "a").entries.first()) toEqualKeyValue (1 to "a")
+
+        fails {
+            expect(mapOf(1 to "a").entries.first()) toEqualKeyValue (1 to "b")
+        }
+    }
+
+    @Test
+    fun keyFeature() {
+        // subject here is of type Int (actually 1)
+        expect(mapOf(1 to "a").entries.first()).key toEqual 1
```

Comment:
IMO this is misleading as one does not know if you mean the initial subject `map(1 to "a")` or the one after `key`. I suggest you re-format it to the following (also in other places):
```suggestion
        expect(mapOf(1 to "a").entries.first()).key toEqual 1
         //                                      |
         //                                      subject is now of type Int (actually 1)
```
And to make it even more obvious, could also add an extra line (up to you which one you prefer)
```suggestion
        expect(mapOf(1 to "a").entries.first()).key toEqual 1
         //          |                           |
         //          |                           subject is now of type Int (actually 1)
         //         subject is of type Map<Int, String>
```